### PR TITLE
Docker multistage build image on commit

### PR DESCRIPTION
**Pull request by valentinab25**

### CODE REVIEW

**Comment by valentinab25** [comment]
Refs #153590 Refs #146850

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        cleanWs(cleanWhenAborted: true, cleanWhenFailure: true, cleanWhenNotBuilt: true, cleanWhenSuccess: true, cleanWhenUnstable: true, deleteDirs: true)
+        script {
+          def url = "${env.BUILD_URL}/display/redirect"
+          def status = currentBuild.currentResult
+          def subject = "${status}: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]'"
+          def summary = "${subject} (${url})"
+          def details = """<h1>${env.JOB_NAME} - Build #${env.BUILD_NUMBER} - ${status}</h1>
+                           <p>Check console output at <a href="${url}">${env.JOB_BASE_NAME} - #${env.BUILD_NUMBER}</a></p>
+                        """
+          emailext (subject: summary, body: details, attachLog: true, compressLog: true, recipientProviders: [[$class: 'DevelopersRecipientProvider'], [$class: 'CulpritsRecipientProvider']])
```
Comment (juanluisrp) [inline Jenkinsfile]: The email list here seems to be too large. Maybe you could reduce the list of recipients?

Comment (valentinab25) [reply Jenkinsfile]: It sends the email to the person that did the commit only.

Comment (juanluisrp) [reply Jenkinsfile]: Ok, maybe it was just the first build the one that sent the email to any that had committed to the project ever.

Comment (fxprunayre) [reply Jenkinsfile]: No, I just received one email with CC to other GeoNetwork developers (who don't need to be notified of changes in here)
image

So can you limit the mail to juan and me ? Thanks.

**Pull request merged**